armbian/base/scripts/bbb-systemctl.sh

```diff
@@ -6,24 +6,26 @@ set -eu
 
 function usage() {
     echo "BitBox Base: batch control system units"
-    echo "Usage: bbb-systemctl <status|start|restart|stop>"
+    echo "Usage: bbb-systemctl <status|start|restart|stop|enable|disable>"
 }
 
 ACTION=${1:-"status"}
 
-if ! [[ ${ACTION} =~ ^(status|start|restart|stop|enable|disable)$ ]]; then
-	usage
-	exit 1
+if [[ ${ACTION} == "-h" ]] || [[ ${ACTION} == "--help" ]]; then
+  usage
+	exit 0
```

Comment:
Nit: Mixed indentation (hard tab). We should use the same indentation consistently, which seems to be two spaces for `.sh` scripts.

(And elsewhere.)